components/engine_panic/src/import.rs

```diff
@@ -27,7 +27,16 @@ impl IngestExternalFileOptions for PanicIngestExternalFileOptions {
     fn new() -> Self {
         panic!()
     }
+
     fn move_files(&mut self, f: bool) {
         panic!()
     }
+
+    fn get_write_global_seqno(&mut self) -> bool {
```

Comment:
```suggestion
    fn get_write_global_seqno(self) -> bool {
```

same for other call sites.

Reply:
Will apply the change once https://github.com/tikv/rust-rocksdb/pull/633 is merged.